test/steps/data.go

```diff
@@ -31,6 +31,7 @@ type Data struct {
 	StartTime              time.Time
 	KogitoExamplesLocation string
 	ScenarioName           string
+	ScenarioContext        map[string]string
```

Comment:
For simplicity, I set the scenario context as a map of strings. The map[string]interface{} would make us to cast types here and there. Also, if strings is not enough, we can always change it in the future.
What do you think about it?

Reply:
good for me

Reply:
ok for me